custom_components/tahoma/coordinator.py

```diff
@@ -50,16 +55,36 @@ async def _async_update_data(self) -> Dict[str, Device]:
             raise UpdateFailed(f"Error communicating with the TaHoma API: {exception}")
         else:
             for event in events:
+                if event.name == "DeviceAvailableEvent":
+                    self.devices[event.deviceurl].available = True
+
                 if event.name == "DeviceStateChangedEvent":
                     for state in event.device_states:
                         device = self.devices[event.deviceurl]
                         if state.name not in device.states:
                             device.states[state.name] = state
                         device.states[state.name].value = self._get_state(state)
+
+                if (
+                    event.name == "ExecutionRegisteredEvent"
+                    and event.exec_id in self.executions
+                ):
+                    self.update_interval = timedelta(seconds=1)
+
+                if (
```

Comment:
The braces are useless, you only use the and operator, their is no priority issue.

Reply:
The braces are added by black.. I believe because it is a multi line if statement, due to the amount of characters.